pegasus/sites.v3/code.org/public/css/common.css

```diff
@@ -396,7 +396,7 @@
   margin: 10px 10px 0 0;
 }
 .share-buttons button i {
-  padding-right: .5em;
+  margin-right: .5em;
```

Comment:
Updates spacing on the share buttons as seen on https://code.org/promote:
<img width="500" alt="Screenshot 2023-05-18 at 4 02 51 PM" src="https://github.com/code-dot-org/code-dot-org/assets/9256643/32175d77-915d-4a8f-9dc4-6da69aa54e07">

Reply:
Is it just me, or is the spacing between the Twitter and Download buttons smaller than the spacing between the FB / Twitter buttons and the Download / Stats buttons?

Reply:
Re: the Facebook icon - this is one instance where the icon is super narrow compared to the bounding box — also that is an outdated icon so I updated it w/ the newer version! 

#### Old
<img width="258" alt="Screenshot 2023-05-31 at 12 39 09 PM" src="https://github.com/code-dot-org/code-dot-org/assets/9256643/d1cd6544-6636-4875-9c62-e00db1ff423b">

#### New 
<img width="315" alt="Screenshot 2023-05-31 at 12 43 41 PM" src="https://github.com/code-dot-org/code-dot-org/assets/9256643/829b1cf6-97e4-4dde-a20d-453d4e369d1d">
<img width="740" alt="Screenshot 2023-05-31 at 12 43 49 PM" src="https://github.com/code-dot-org/code-dot-org/assets/9256643/3af96940-43c2-4861-9da6-675a79069dfb">

---

For the Download presentation button, it looks like there's an extra space added before " Download" and I'm not sure how to fix that. This may require another separate refactor PR to address translatability, rtl language styles, and accessibility — I'll make a note of it [in this ticket](https://codedotorg.atlassian.net/browse/DES-272?atlOrigin=eyJpIjoiZTllNzRiNTBlYzMyNGNkOWE5ZTExNTk4MWU4NWYxZGQiLCJwIjoiaiJ9). 
<img width="314" alt="Screenshot 2023-05-31 at 12 44 55 PM" src="https://github.com/code-dot-org/code-dot-org/assets/9256643/8c0f06f8-7954-43ee-a77e-75f2c3cc76ef">

Reply:
Ah, my fault, I thought this spacing was referring to the between the buttons themselves, not between the text and the icon. Could this be padding instead of margin-right for RTL stuff?

Reply:
Oh you're right! The two sets of buttons are each in their own container (socials are together, and download + more stats are together) which is causing the issue. It also looks like the social buttons are hidden on mobile. I think this entire partial can be improved ✨ 

Still resulted in an improvement to the FB icon though 😅 🎉

Reply:
Yep, agreed! Thanks for explaining